.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
what is this file for?

Reply:
I am not exactly sure how this was created. It's a way to set up how the code will be run. I deleted the file.